terraform/service_cleanup_export.tf

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Create and deploy the service
+#
+
+resource "google_service_account" "cleanup-export" {
+  project      = data.google_project.project.project_id
+  account_id   = "en-cleanup-export-sa"
+  display_name = "Exposure Notification Cleanup Export"
+}
+
+resource "google_project_iam_member" "cleanup-export-cloudsql" {
+  project = data.google_project.project.project_id
+  role    = "roles/cloudsql.client"
+  member  = "serviceAccount:${google_service_account.cleanup-export.email}"
+}
+
+resource "google_secret_manager_secret_iam_member" "cleanup-export-db-pwd" {
+  provider = google-beta
+
+  secret_id = google_secret_manager_secret.db-pwd.id
+  role      = "roles/secretmanager.secretAccessor"
+  member    = "serviceAccount:${google_service_account.cleanup-export.email}"
+}
+
+resource "google_storage_bucket_iam_member" "cleanup-export-objectadmin" {
+  bucket = google_storage_bucket.export.name
+  role   = "roles/storage.objectAdmin"
+  member = "serviceAccount:${google_service_account.cleanup-export.email}"
+}
+
+resource "google_cloud_run_service" "cleanup-export" {
+  name     = "cleanup-export"
+  location = var.region
+
+  template {
+    spec {
+      service_account_name = google_service_account.cleanup-export.email
+
+      containers {
+        image = "us.gcr.io/${data.google_project.project.project_id}/github.com/google/exposure-notifications-server/cmd/cleanup-export:latest"
+
+        dynamic "env" {
+          for_each = local.common_cloudrun_env_vars
+          content {
+            name  = env.value["name"]
+            value = env.value["value"]
+          }
+        }
+      }
+    }
+
+    metadata {
+      annotations = {
+        "run.googleapis.com/cloudsql-instances" : google_sql_database_instance.db-inst.connection_name
+        "autoscaling.knative.dev/maxScale" : "5"
```

Comment:
@mikehelmick please advise

Reply:
it's fine, but probably unnecessary.

the cleanup jobs require authentication by the scheduler SA.

Reply:
Updated